app/services/import/legacy-import/licence-versions.mapper.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+/**
+ * Maps the import licence versions data to the desired format
+ * @module LegacyImportLicenceVersionMapper
+ */
+
+const { formatStandardDateToISO } = require('../../../lib/dates.lib.js')
+
+const statuses = {
+  CURR: 'current',
+  SUPER: 'superseded',
+  DRAFT: 'draft'
+}
+
+const createExternalId = (licenceVersion) => {
+  const { FGAC_REGION_CODE, AABL_ID, ISSUE_NO, INCR_NO } = licenceVersion
+
+  return `${FGAC_REGION_CODE}:${AABL_ID}:${ISSUE_NO}:${INCR_NO}`
+}
+
+/**
+ * Maps the import licence versions data to the desired format
+ *
+ * @param {LegacyLicenceVersionsArray} licenceVersions
+ * @returns {}
+ */
+function go (licenceVersions) {
+  return _mapLicenceVersions(licenceVersions)
+}
+
+function _mapLicenceVersions (licenceVersions) {
+  return licenceVersions.map((licenceVersion) => {
+    const issue = licenceVersion.ISSUE_NO // mapped to the legacy purpose id -
+    const increment = licenceVersion.INCR_NO
+
+    return {
+      createdAt: new Date().toISOString(),
+      endDate: formatStandardDateToISO(licenceVersion.EFF_END_DATE),
+      externalId: createExternalId(licenceVersion),
+      increment: Number(increment),
+      issue: Number(issue),
+      startDate: formatStandardDateToISO(licenceVersion.EFF_ST_DATE),
+      status: statuses[licenceVersion.STATUS],
+      updatedAt: new Date().toISOString(),
+      purposes: _mapPurposes(licenceVersion)
+    }
+  })
+}
+
+function _mapPurposes (licenceVersion) {
+  return licenceVersion.purposes.map((purpose) => {
+    return mapLicencePurpose(purpose)
+  })
+}
+
+module.exports = {
+  go
+}
+
+const mapLicencePurpose = (purpose) => {
+  // double check no longer needed because of update sql
```

Comment:
// double check no longer needed because of update sql
  // issue: purpose.AABV_ISSUE_NO,
  //   increment: purpose.AABV_INCR_NO,